Artifacts/windows-certificate-from-keyvault/Artifactfile.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title":  "Certificate from Azure Key Vault",
+    "publisher": "Greg Pakes",
+    "description": "Installs a certificate stored in Azure Key Vault.",
+    "tags": [
+        "Windows",
+        "Certificate",
+        "Key Vault"
+    ],
+    "parameters": {
+        "vaultName": {
+        "type": "string",
+        "displayName": "Vault Name",
+        "description": "Name of the Azure Key Vault"
+        },
+        "secretName": {
+        "type": "string",
+        "displayName": "Secret Name",
+        "description": "Name of the secret within the Azure Key Vault"
+        },
+        "azureServicePrincipalClientId": {
+        "type": "string",
+        "displayName": "Azure Service Principal Client Id",
+        "description": "The client Id of the Service Principal used to access Azure"
+        },
+        "azureServicePrincipalKey": {
+        "type": "securestring",
+        "displayName": "Azure Service Principal Key",
+        "description": "The key of the Azure Service Principal"
+        },
+        "azureServicePrincipalTenantId": {
+        "type": "string",
+        "displayName": "Azure Service Tenant Id",
+        "description": "The tenant id of the Azure Service Principal"
+        }
+    },
+    "targetOsType": "Windows",
+    "runCommand": {
+        "commandToExecute": "[concat('powershell.exe -NonInteractive -ExecutionPolicy bypass \"& ./InstallCertificate.ps1 -vaultName ', parameters('vaultName'), ' -secretName ', parameters('secretName'), ' -azureServicePrincipalClientId ', parameters('azureServicePrincipalClientId'), ' -azureServicePrincipalKey ', parameters('azureServicePrincipalKey'), ' -azureServicePrincipalTenantId ', parameters('azureServicePrincipalTenantId'), '\"')]"
```

Comment:
Fixed in https://github.com/Azure/azure-devtestlab/pull/248/commits/42f85103f9c4e29f4e1c16c767eeeed8689bcdf2